src/notification.h

```diff
@@ -2,6 +2,13 @@
 #ifndef DUNST_NOTIFICATION_H
 #define DUNST_NOTIFICATION_H
 
+//TODO: Remove cyclic dependency
+typedef struct _notification_colors {
+        char *frame;
+        char *bg;
+        char *fg;
+} NotificationColors;
```

Comment:
While `Actions` and `RawImage` are using CamelCase, I'd prefer if we shifted to camel_case for most things and slowly start shifting the codestyle that way. Though fixing everything right now is not a priority I'd prefer if we didn't introduce new stuff like this.

Reply:
This fits perfect into #483